openwisp_ipam/tests/test_api.py

```diff
@@ -251,6 +251,24 @@ def test_hosts_list_api(self):
         self.assertEqual(self.client.get(response.data['previous']).status_code, 200)
         self.assertIsNone(response.data['next'])
 
+    def test_bearer_auth(self):
+        self._create_user(username='tester', password='tester')
+        params = {'username': 'tester', 'password': 'tester'}
+        url = reverse('users:api-token-auth')
```

Comment:
@DaffyTheDuck the name of the view is `user_token_auth` and not `api-token-auth` please see it here: https://github.com/openwisp/openwisp-users/blob/master/openwisp_users/api/urls.py#L14